workflow/automation/execution_scripts/queue_monitor.py

```diff
@@ -60,6 +60,13 @@ def get_queue_entry(
         status=data_dict[MgmtDB.col_status],
         job_id=data_dict[MgmtDB.col_job_id],
         error=data_dict.get("error"),
+        queued_time=data_dict.get(MgmtDB.col_queued_time),
+        start_time=data_dict.get(MgmtDB.col_start_time),
+        end_time=data_dict.get(MgmtDB.col_end_time),
+        nodes=data_dict.get(MgmtDB.col_nodes),
+        cores=data_dict.get(MgmtDB.col_cores),
+        memory=data_dict.get(MgmtDB.col_memory),
+        wct=data_dict.get(MgmtDB.col_wct),
```

Comment:
This makes me a little sad, but I also think trying to do val=dict.get(col_val) programmatically would be a bit overkill

Reply:
Use a dict comprehension and then unpack it when passing to init?